### PR TITLE
fix: descriptions buttom right padding style

### PR DESCRIPTION
**Pull request by crazyair**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/49671

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     fix descriptions buttom right padding style      |
| 🇨🇳 Chinese |    修复描述组件间距样式       |

### CODE REVIEW

File: components/descriptions/demo/text.tsx [hunk 1]
```diff
@@ -113,6 +113,13 @@ const items: DescriptionsProps['items'] = [
     label: 'Config Info',
     children: <Table size="small" pagination={false} dataSource={dataSource} columns={columns} />,
   },
+  {
+    key: '14',
+    label: 'long',
+    span: 1,
+    children:
+      'looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooog',
```
Comment (zombieJ) [inline components/descriptions/demo/text.tsx:121]: 这个不要这么随意，写的有意义的东西吧

File: components/descriptions/style/index.ts [hunk 2]
```diff
@@ -28,6 +28,11 @@ export interface ComponentToken {
    * @descEN Bottom padding of item
    */
   itemPaddingBottom: number;
+  /**
+   * @desc 子项右间距
+   * @descEN Right padding of item
+   */
+  itemPaddingRight: number;
```
Comment (zombieJ) [inline components/descriptions/style/index.ts:35]: @MadCcc 确定一下要不要加这个

File: components/descriptions/demo/text.tsx (hunk 1 above)
Comment (crazyair) [reply components/descriptions/demo/text.tsx:121]: 这个是 debug 的，还好

File: components/descriptions/style/index.ts (hunk 2 above)
Comment (crazyair) [reply components/descriptions/style/index.ts:35]: 这个看起来，RTL 下 right 要改成 left

image


image

Comment (crazyair) [reply components/descriptions/style/index.ts:35]: 改成 paddingInlineEnd 了

**Review by MadCcc — DISMISSED** [review]
LGTM

**Comment by afc163** [comment]
体积大了好多。

**Review by zombieJ — DISMISSED** [review]

**Review by MadCcc — APPROVED** [review]

**Pull request merged**